pkg/app/execontext.go

```diff
@@ -207,3 +215,13 @@ func ShowCommunityInfo() {
 	fmt.Printf("docker-slim: message='join the Discord server to ask questions or to share your feedback' info='%s'\n", consts.CommunityDiscord)
 	fmt.Printf("docker-slim: message='Github discussions' info='%s'\n", consts.CommunityDiscussions)
 }
+
+func JSONFormatter(data string) {
+
+	var out io.Writer
+	enc := json.NewEncoder(out)
+	enc.SetIndent("", "    ")
```

Comment:
we need to have single line json records instead of pretty printed json

Reply:
Tried to fix this